.github/workflows/scripts/update-major-version-tag.sh

```diff
@@ -28,4 +28,4 @@ git tag "$MAJOR_VERSION_TAG" HEAD
 git push --tags
 echo "Updated shorthand major version tag."
 
-echo "::set-output name=MAJOR_VERSION_TAG::$MAJOR_VERSION_TAG"
+echo "MAJOR_VERSION_TAG=$MAJOR_VERSION_TAG" >> $GITHUB_OUTPUT
```

Comment:
Nit: We should probably follow the lint suggestion here:
```suggestion
echo "MAJOR_VERSION_TAG=$MAJOR_VERSION_TAG" >> "$GITHUB_OUTPUT"
```

As you pointed out elsewhere, this is how the GitHub docs suggest doing it, so it's probably safe to assume that they're setting "$GITHUB_OUTPUT" to a filename with no spaces in it that would cause word splitting. But still there's no harm in protecting against word splitting anyway